Doxyfile

```diff
@@ -917,7 +934,7 @@ EXCLUDE_SYMLINKS       = NO
 # Note that the wildcards are matched against the file with absolute path, so to
 # exclude all test directories for example use the pattern */test/*
 
-EXCLUDE_PATTERNS       =
+EXCLUDE_PATTERNS       = */inc/_*.h */tests/* */test/* */internal/* *_private.h
```

Comment:
Add *_internal.h since we have a few of those as well.

```suggestion
EXCLUDE_PATTERNS       = */inc/_*.h */tests/* */test/* */internal/* *_private.h *_internal.h
```